sdks/python/apache_beam/io/gcp/datastore/v1/query_splitter.py

```diff
@@ -37,7 +37,7 @@
                            PropertyFilter.GREATER_THAN,
                            PropertyFilter.GREATER_THAN_OR_EQUAL]
 except ImportError:
-  UNSUPPORTED_OPERATORS = None
+  UNSUPPORTED_OPERATORS = None  # type: ignore
```

Comment:
Are all of these due to trying to suppress warnings?

Reply:
Correct.  I can add a comment to the code clarify.  

The problem is that if we mark `UNSUPPORTED_OPERATORS` as `Optional` then mypy will generate numerous errors in this module, because the code assumes the import succeeded and that `UNSUPPORTED_OPERATORS` is _not_ `None`.   

An alternative would be to set it to `[]`.

Reply:
Is it possible to opt certain modules out, if they become to noisy (and adding all the suppression is noisy as well)?

Reply:
Yes, you can exclude errors from specific modules and packages.  That said, it's still a lot of work to get even a restricted set of modules passing, and if you restrict the set of checked modules too much then you undermine the value of type checking.  I've written a [mypy runner](https://github.com/chadrik/mypy-runner) that let's you focus on a specific set of errors, so that you can tackle problems one error _type_ at a time, rather than one file at a time.  We're using this where I work and it's been quite successful.